login-system/sql/sql_import.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+//connection variables
+$host = 'localhost';
+$user = 'root';
+$password = 'passwd';
```

Comment:
you have defined db.php so why not include it here.

Reply:
these are the sql commands for creating database, tables, etc. and db.php is for connecting to database